components/camel-pg-replication-slot/src/main/java/org/apache/camel/component/pg_replication_slot/PgReplicationSlotComponent.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.pg_replication_slot;
```

Comment:
just a personal opinion. i don't like underscores in package names. by name, it is hard to think of one. i would replace underscores with dots which would give more directory depth. i think it is something nicer. don't have a reason, though

Reply:
What about `.pg.replicationslot`?

Reply:
yes good. thanks for doing it.